notebooks/standardize_events_data.ipynb

```diff
@@ -1071,27 +1070,24 @@
     "                'receiver': data_list[2],\n",
     "                'timestamp': data_list[3]\n",
     "            })\n",
-    "        else:\n",
-    "            return pd.Series()\n",
+    "       else:\n",
+    "            raise ValueError(\"Unrecognized key_name\") # Raising an error for unrecognized key_name\n",
     "    except Exception as e:\n",
     "        # Handle any parsing errors\n",
-    "        return pd.Series()\n",
+    "        raise RuntimeError(f\"Failed to extract data from row: {str(e)}\") from e\n",
     "\n",
     "# Filter the dataset for 'hashstack_v1' protocol\n",
     "hashstack_v1_data = df_events[df_events['Protocol'] == 'hashstack_v1']\n",
     "\n",
     "# Apply the extraction function to each row\n",
-    "extracted_elements = hashstack_v1_data.apply(extract_hashstack_v1, axis=1)\n",
+    "standardized_hashstack_v1_events = hashstack_v1_data.apply(standardize_hashstack_v1_event, axis=1)\n",
```

Comment:
I'm getting `RuntimeError: Failed to extract data from row: Unrecognized key_name` here.